dspace/config/modules/publication-lookup.cfg

```diff
@@ -1 +1 @@
-publication.url = ${publication.url}
\ No newline at end of file
+publication-lookup.publication.url = ${publication.url}
```

Comment:
Is there a reason that this defaults to a non-existent setting? `publication.url` doesn't exist in any other configuration file, so this means the default value will always be null.  Is there a more appropriate default?

I guess I'm wondering whether this should be set to `*` by default (which I think is the recommended setting?), instead of requiring users to add a new `publication.url` field elsewhere.